src/SoundList.js

```diff
@@ -9,7 +9,35 @@ function timeConvert(minutes) {
   return `${(min < 10 ? "0" : "") + min}:${secs < 10 ? "0" : ""}${secs}`;
 }
 
-const SoundList = ({ tracks, header, onSoundClick, className = "" }) => {
+// Psuedo code:
+// function highlightSound(track, sound) {
+//   let packSoundArr = pack sounds array;
```

Comment:
The `<SoundList />` component doesn't actually know what the sounds in the Pack list and the Similar list are. It just receives `tracks` as a prop from a parent component in the tree.

Reply:
I have a couple of questions:
     ` const selectedTrackClassName` - does this refer to the track inside of the SoundList component?
      `const selectedSoundDataClassName` - does this refer to the selected sound on the Sound Page?

Reply:
I answered the question below, since I think it's easier to have the discussion near the lines in question.